src/main/java/io/quarkus/bot/release/step/AnnounceRelease.java

```diff
@@ -82,7 +82,7 @@ public int run(Context context, Commands commands, GitHub quarkusBotGitHub, Rele
                     comment.append("  * Update requirements if needed\n");
                 }
             }
-            if (Branches.isLts(releaseInformation.getBranch())) {
+            if (Branches.isLts(releaseInformation.getBranch()) && !releaseInformation.isMaintenance()) {
```

Comment:
I think I would rather test `releaseInformation.isFirstFinal()` to be consistent with the rest but feel free to push back if you have a strong feeling.